test/helpers.bash

```diff
@@ -195,3 +163,29 @@ calculate_pods_per_node() {
     echo "75"
   fi
 }
+
+verify_object_count(){
+  local resource=$1
+  local count=$2
+  local namespace=$3
+  local selector=$4
+  local fieldSelector=$5
+  local CMD="oc get $1 --no-headers"
+  if [[ -n ${namespace} ]]; then
+    CMD+=" -n ${namespace}"
+  else
+    CMD+=" -A"
+  fi
+  if [[ -n ${selector} ]]; then
+    CMD+=" -l ${selector}"
+  fi
+  if [[ -n ${fieldSelector} ]]; then
+    CMD+=" --field-selector ${fieldSelector}"
+  fi
+  echo "${CMD}"
+  counted=$(${CMD} | wc -l)
```

Comment:
I had an issue in the past where using `wc -l` didn't work for me when the expected value was `0`.
So, instead, I used
```
kubectl get <resource> -o json | jq '.items | length'
```
See: https://github.com/kube-burner/kube-burner/blob/2347521b57f78587e5d60f8260135cdea0bb89b2/test/helpers.bash#L167